server:
  base_url: "http://127.0.0.25:5000"
  username: "team_user"
  password: "team_pass"
  team_id: 1
  telemetry_rate: 1.0  # Hz